src/content/templates/graph-use_cpu.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
this is graph-cpu the metrics come from the cpu plugin. if these are new graphs, add them to the existing graph-cpu template.